README.md

```diff
@@ -1,602 +1,387 @@
-specs and cucumber features, and make patches.
-
-NOTE: You do not need to use rspec-dev to work on a specific RSpec repo. You
-can treat each RSpec repo as an independent project.
-Please see the following files:
-
-For `rspec-rails`-specific development information, see
-
-- [Build details](BUILD_DETAIL.md)
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Detailed contributing guide](CONTRIBUTING.md)
-- [Development setup guide](DEVELOPMENT.md)
-
-
-## Model Specs
-
-Use model specs to describe behavior of models (usually ActiveRecord-based) in
-the application.
-
-Model specs default to residing in the `spec/models` folder. Tagging any
-context with the metadata `:type => :model` treats its examples as model
-specs.
-
-For example:
-
-```ruby
-require "rails_helper"
-
-RSpec.describe Post, :type => :model do
-  context "with 2 or more comments" do
-    it "orders them in reverse chronologically" do
-      post = Post.create!
-      comment1 = post.comments.create!(:body => "first comment")
-      comment2 = post.comments.create!(:body => "second comment")
-      expect(post.reload.comments).to eq([comment2, comment1])
-    end
-  end
-end
+# Run a single example from a spec file (by line number)
+$ bundle exec rspec spec/controllers/accounts_controller_spec.rb:8
 ```
 
-For more information, see [cucumber scenarios for model
-specs](https://www.relishapp.com/rspec/rspec-rails/docs/model-specs).
-
-## Request Specs
-
-Use request specs to describe the client-facing behavior of the application —
-specifically, the HTTP response to be issued for a given request (a.k.a.
-integration tests). Since such client-facing behavior encompasses controller
-actions, this is the type of spec to use for controller testing.
+## RSpec DSL Basics (or, how do I write a spec?)
 
-Request specs default to residing in the `spec/requests`, `spec/api`, and
-`spec/integration` directories. Tagging any context with the metadata `:type =>
-:request` treats its examples as request specs.
-
-Request specs mix in behavior from
-[ActionDispatch::Integration::Runner](http://api.rubyonrails.org/classes/ActionDispatch/Integration/Runner.html),
-which is the basis for [Rails' integration
-tests](http://guides.rubyonrails.org/testing.html#integration-testing).
+In RSpec, application behavior is described
+**first in (almost) plain English, then again in test code**, like so:
 
 ```ruby
-require 'rails_helper'
-
-RSpec.describe "home page", :type => :request do
-  it "displays the user's username after successful login" do
-    user = User.create!(:username => "jdoe", :password => "secret")
-    get "/login"
-    assert_select "form.login" do
-      assert_select "input[name=?]", "username"
-      assert_select "input[name=?]", "password"
-      assert_select "input[type=?]", "submit"
+RSpec.describe Member do  #
+  context 'in death' do   # (almost) plain English
+    it 'has a name' do    #
+      expect(dead_member.name).to eq('Robert Paulson')  # test code
```

Comment:
Two things, please don't use constant names in describe blocks, and please don't use a persons name.

Reply:
Fixed–but when you generate spec files with `rails g`, isn't that how they're formatted?